docker/entrypoint.sh

```diff
@@ -59,7 +49,5 @@ else
   export SUPERVISORD_CADDY=true
 fi
 
-chown -R www-data:www-data /pelican-data/.env /pelican-data/database
```

Comment:
This should stay as if for some reason the perms change a new key will be generated and the db will be nuked

Reply:
You can't, because the container is run as `www-data`, so it lacks permission to change ownership. If the permissions change, it can't overwrite the files anyways, so it shouldn't matter? For example, if .env is restricted and can't be read so it decides to make a new key, writing .env will also fail anyways? Is there something I'm missing?